db/system_distributed_keyspace.cc

```diff
@@ -54,9 +61,37 @@ schema_ptr view_build_status() {
     return schema;
 }
 
+schema_ptr cdc_topology_description() {
+    thread_local auto schema = [] {
+        auto id = generate_legacy_id(system_distributed_keyspace::NAME, system_distributed_keyspace::CDC_TOPOLOGY_DESCRIPTION);
+        return schema_builder(system_distributed_keyspace::NAME, system_distributed_keyspace::CDC_TOPOLOGY_DESCRIPTION, {id})
+                .with_column("time", timestamp_type, column_kind::partition_key)
+                .with_column("description", bytes_type)
+                .with_column("expired", timestamp_type)
+                .with_version(system_keyspace::generate_schema_version(id))
+                .build();
+    }();
+    return schema;
```

Comment:
Please add to docs/ in CQL format, see for example tracing.md, with a discussion about what each column means.

Reply:
Added docs/cdc.md